### PR TITLE
Chore/adapt models frontend: Support new models from BatteryPass, DigitalProductPass and TransmissionPass

### PR DESCRIPTION
**Pull request by saudkhan116**
# Why we create this PR?
 
Adapt Frontend componet with the following new models:

- Digital Product Pass v5.0.0
- Battery Pass 6.0.0
- Transmission Pass 3.0.0

# What we want to achieve with this PR?
 
To support new model versions in the dpp-frontend
 
# What is new?
 
## Updated
- Updated frontend component to support new data model versions:
    - Digital Product Pass v5.0.0
    - Battery Pass 6.0.0
    - Transmission Pass 3.0.0

## PR Linked to:
#270

### CODE REVIEW

File: dpp-frontend/package.json [hunk 1]
```diff
@@ -27,7 +28,7 @@
     "vite": ">=4.5.2",
     "vite-plugin-vuetify": "^1.0.2",
     "vue": "^3.2.47",
-    "vue-chartjs": "^5.3.0",
+    "vue-chartjs": "^5.3.1",
     "vue-i18n": "9.2.2",
     "vue-router": "^4.0.13",
```
Comment (matbmoser) [inline dpp-frontend/package.json:33]: IP Checks needs to be runned here!

File: dpp-backend/scripts/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+requests==2.31.0
```
Comment (matbmoser) [inline dpp-backend/scripts/requirements.txt:1]: We need Ip check here

**Review by matbmoser — APPROVED** [review]

File: dpp-backend/scripts/requirements.txt (hunk 2 above)
Comment (matbmoser) [reply dpp-backend/scripts/requirements.txt:1]: [DEPENDENCIES_BACKEND_SCRIPTS.txt](https://github.com/user-attachments/files/15902785/DEPENDENCIES_BACKEND_SCRIPTS.txt)

Comment (matbmoser) [reply dpp-backend/scripts/requirements.txt:1]: you can use it and remove the txt

**Review by matbmoser — CHANGES_REQUESTED** [review]
Click the wrong button, IP checks needed

File: dpp-frontend/package.json (hunk 1 above)
Comment (saudkhan116) [reply dpp-frontend/package.json:33]: IP checks executed

**Review by matbmoser — APPROVED** [review]
Approved!

**Pull request merged**